cli/src/revset_util.rs

```diff
@@ -107,6 +107,31 @@ impl<'repo> RevsetExpressionEvaluator<'repo> {
     }
 }
 
+fn warn_user_redefined_builtin(
+    ui: &Ui,
+    source: &ConfigSource,
+    name: &str,
+) -> Result<(), CommandError> {
+    match source {
+        ConfigSource::Env | ConfigSource::User | ConfigSource::Repo | ConfigSource::CommandArg => {
```

Comment:
nit: (at your option) I would add early return `if source == ConfigSource::Default`, but exhaustive match seems also good.

Reply:
I've left the exhaustive match, but I've moved the `ConfigSource::Default` case to the top, so it's easier to see that we bail there with a noop.